### PR TITLE
Small fixes and polish

### PR DESCRIPTION
**Pull request by beets**
- Add URL for the sub-pages to "reset" apps
- Update text in get_involved and minor copy edits elsewhere
- Timeline -> Timelines
- Update Google Font included and actually use it for the header
- Move static content templates to a subfolder
- Fix typo in README

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ git push -u origin new_branch_name
 ```
 
 Then in your forked repo, you can send a Pull Request. If this is your first
-time contributing to a Google Open Source project, you may need to follow the
+time contributing to a Google Open oource project, you may need to follow the
```
Comment (shifucun) [inline README.md:46]: source

File: server/templates/base.html [hunk 2]
```diff
@@ -12,8 +12,8 @@
  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  See the License for the specific language governing permissions and
  limitations under the License.
-#}
-{#
+j#}
```
Comment (shifucun) [inline server/templates/base.html:15]: remove "j"?

File: README.md (hunk 1 above)
Comment (beets) [reply README.md]: done

File: server/templates/base.html (hunk 2 above)
Comment (beets) [reply server/templates/base.html]: done

**Comment by beets** [comment]
thanks for the catch!

**Pull request merged**